### ISSUE TITLE
Profile dialog in community: make languages multi-select

### ISSUE THREAD

**Issue by corinnewo** [ji-devs/ji-cloud#3036]
In the profile dialog:

1. Change the language field to a multi-select field, and list the languages that this person speaks
2. In the publish dialog, 

- if there is only one language, then that becomes the default language of the JIG.
- If more than one language, then leave the default blank to avoid unnecessary wrong data.

![image](https://user-images.githubusercontent.com/16290361/183385623-1d404fd9-5afa-43a3-89f3-8d338c90fa47.png)

**Comment by corinnewo** [comment]
Also connected with #3026

**Comment by MendyBerger** [comment]
@corinnewo if we change Language to a multi select we also need to figure out the how to fit that in to the signup process

**Comment by corinnewo** [comment]
@MendyBerger This language is not connected with the signup process at all. In signup, it is the language that you want the newsletters to be sent in. In profile, it is defining the languages that you speak.

**Comment by MendyBerger** [comment]
So I think we need to make that clear in the signup process.

Something else to consider, that means that most users won't have any languages, since they'll have to go to their profile to add them in

**Comment by MendyBerger** [comment]
Blocked on #3053

**Comment by MendyBerger** [comment]
Resolved

**Comment by MendyBerger** [comment]
@corinnewo mind opening a separate issue to resolve the newsletter language?
